recipes/cp2k/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cp /home/conda/conda-recipes/cp2k/Linux-x86-64-conda.sopt arch/Linux-x86-64-conda.sopt
+cd makefiles
+make -j${CPU_COUNT} ARCH=Linux-x86-64-conda VERSION=sopt
+cp ../exe/Linux-x86-64-conda/cp2k.sopt ${PREFIX}/bin/cp2k.sopt
+cp ../exe/Linux-x86-64-conda/cp2k_shell.sopt ${PREFIX}/bin/cp2k_shell.sopt
```

Comment:
Is there a way to run tests?

Reply:
I added `make -j${CPU_COUNT} ARCH=Linux-x86-64-conda VERSION=sopt test`